easybuild/easyconfigs/m/matplotlib/matplotlib-3.3.3-cpeGNU-20.10.eb

```diff
@@ -0,0 +1,60 @@
+# contributed by Luca Marsella (CSCS)
+easyblock = 'PythonBundle'
+
+name = 'matplotlib'
+version = '3.3.3'
+
+homepage = 'https://matplotlib.org'
+description = """matplotlib is a python 2D plotting library which produces publication quality figures in a variety of
+ hardcopy formats and interactive environments across platforms. matplotlib can be used in python scripts, the python
+ and ipython shell, web application servers, and six graphical user interface toolkits."""
+
+toolchain = {'name': 'cpeGNU', 'version': '20.10'}
+toolchainopts = {'pic': True, 'verbose': False}
+
+dependencies = [
+    ('cray-mpich', EXTERNAL_MODULE),
```

Comment:
Is `cray-mpich` needed here?

Reply:
Probably not, as far as I can see on https://matplotlib.org/users/installing.html. I can remove the dependency